src/dmd/statement.d

```diff
@@ -1591,6 +1591,15 @@ extern (C++) final class SwitchStatement : Statement
 
     final bool checkLabel()
     {
+        /*
+        * Checks the scope of a label for existing variable declaration.
+        *
+        * Params:
+        *   vd = variable declaration to check
+        *
+        * Returns: `true` if the variables declared in this label would be skipped
+                   `false` otherwise
+        */
```

Comment:
Seems a bit verbose, especially for a non-ddoc symbol. Does  `Params` really add anything?

Reply:
I think it _should_ have a Ddoc comment.

Reply:
The "`false` otherwise" is redundant and can be removed.